datafusion/physical-expr/src/math_expressions.rs

```diff
@@ -290,6 +295,14 @@ pub fn round(args: &[ArrayRef]) -> Result<ArrayRef> {
     }
 }
 
+fn round_downcast_decimal_places(decimal_place: i64) -> Result<i32> {
```

Comment:
I noticed most small helpers inside execution are macros to guarantee inline for performance, should we make this function macro?

Reply:
Thanks, I made it as a macro instead!

Reply:
You could inline a function too. Not necessary to be macros actually as macros are more difficult to read.